pulsar-common/src/main/java/org/apache/pulsar/common/naming/TopicName.java

```diff
@@ -288,6 +288,13 @@ public boolean isGlobal() {
         return cluster == null || Constants.GLOBAL_CLUSTER.equalsIgnoreCase(cluster);
     }
 
+    public String getSchemaName() {
+        return getProperty()
+            + "_" + getCluster()
```

Comment:
Cluster has been removed from topic name. Yes, the change is overall still in progress, though the main idea is that v2 topic names will never have that. 

Schema shouldn't be supported on v1 topics (or API), but in any case we could just do the check, here in `TopicName` class to validate whether cluster should be part of the name or not.

Reply:
done.